src/Symfony/Component/HttpClient/CHANGELOG.md

```diff
@@ -5,3 +5,4 @@ CHANGELOG
 -----
 
  * added the component
+ * added a mock client
```

Comment:
no need to add this in the changelog, as there is no component at all in previous version, so `added the component` describes the changes already.